Docs/articles/Kernel/CGS.md

```diff
@@ -72,38 +74,32 @@ namespace GraphicTest
                 pen.Color = Color.PaleVioletRed;
                 canvas.DrawRectangle(pen, 350, 350, 80, 60);
 
-                Console.ReadKey();
-
-                /* Let's try to change mode...*/
-                canvas.Mode = new Mode(800, 600, ColorDepth.ColorDepth32);
-
                 /* A LimeGreen rectangle */
                 pen.Color = Color.LimeGreen;
                 canvas.DrawRectangle(pen, 450, 450, 80, 60);
 
-                Console.ReadKey();
-
-                Stop();
+                Sys.Power.Shutdown();
```

Comment:
Can you add an example of drawing an image? Simply include the image using a manifest resource stream. Would be cool if you could add a section about images or should that be a separate article?

Reply:
I don't have any image that I could use that's < than 1 MB. Do you have one?

Reply:
I think the Graphics Test kernel has a few. Otherwise just download a random one from google.